main.c

```diff
@@ -451,6 +484,10 @@ static const char* get_codes(flag_type_t flag_type)
     case FLAG_TYPE_GENDERFLUID:
         return codes_gfl;
         break;
+   case FLAG_TYPE_ASEXUAL:
```

Comment:
Add empty line before for consistency, also indentations and replace tabs with spaces